test_integration/test_template.py.in

```diff
@@ -136,7 +136,7 @@ class TestIntegration_@test_name@(unittest.TestCase):
 
         """
 
-        if (sys.exc_info() == (None, None, None) and not
+
             cls._keep_files and not
```

Comment:
What happened to the if()?

Reply:
Good catch.  I tried tested the template properly after fixing this.  Seems to work correctly.